### PR TITLE
task/WG-96 add assets to original questionnaire code

### PR DESCRIPTION
**Pull request by nathanfranklin**
## Overview: ##

This PR updates the frontend viewing of RAPID questionnaires in order to show image assets

## Related Jira tickets: ##

* [WG-96](https://jira.tacc.utexas.edu/browse/WG-96)

## Testing Steps: ##

See steps at https://github.com/TACC-Cloud/geoapi/pull/131

## Notes: ##

Note: I also started to incorporate updated js from RAPID but stopped that to focus just on showing assets.  That WIP code is at https://github.com/TACC-Cloud/hazmapper/tree/task/WG-96-add-assets-to-questionairre and the goal would be to just use un-changed js code to enable us to just drop in their changes without extra work making it typescript.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     - name: Setup Node.js for use with actions
       uses: actions/setup-node@v1
       with:
-        node-version: 16.x
+        node-version: 14.x
```
Comment (taoteg) [inline .github/workflows/main.yml:17]: Is the node downgrade required while at the same time bumping the ES target forward from 2015 to 2020? It looks like node 14 is the minimum version for supporting ES2020 but do we need to revert at all?

File: angular/src/app/components/asset-detail/asset-detail.component.ts [hunk 2]
```diff
@@ -35,6 +35,7 @@ export class AssetDetailComponent implements OnInit {
       this.feature = next;
       try {
         let featureSource = this.envService.apiUrl + '/assets/' + this.feature.assets[0].path;
+
```
Comment (taoteg) [inline angular/src/app/components/asset-detail/asset-detail.component.ts:38]: Is this blank line intentional mid-block?

File: angular/src/app/components/feature-metadata/feature-metadata.component.html [hunk 3]
```diff
@@ -2,12 +2,14 @@
...
-        <b>
-            <span> {{ item.key | titlecase }} </span></b
-        >
-    </div>
-    <div class="cell small-8 text-left break">
-        <code>{{ item.value | json }}</code>
-    </div>
+    <ng-container *ngIf="!item.key.startsWith('_hazmapper')">
+        <div class="cell small-4 text-left break">
+            <b
```
Comment (taoteg) [inline angular/src/app/components/feature-metadata/feature-metadata.component.html:7]: This looks a bit off syntactically...

**Review by taoteg — COMMENTED** [review]
Aside from a few syntax tweaks and the question of node version, LGTM!

File: angular/src/app/components/asset-detail/asset-detail.component.ts (hunk 2 above)
Comment (nathanfranklin) [reply angular/src/app/components/asset-detail/asset-detail.component.ts:38]: dropping 👍

File: .github/workflows/main.yml (hunk 1 above)
Comment (nathanfranklin) [reply .github/workflows/main.yml:17]: cool. reverted.  still have a mistmatch between dockerfile and ci but can address that somewhere else.  (also are tests are limited so probably doesn't matter)

**Review by taoteg — APPROVED** [review]
LGTM!

File: angular/src/app/components/feature-metadata/feature-metadata.component.html (hunk 3 above)
Comment (nathanfranklin) [reply angular/src/app/components/feature-metadata/feature-metadata.component.html:7]: hmm, predates this PR but in the diff as I'm moving it in a block to conditionally skip it if key is _hazmapper (that is where some non-user data is stored about the assets in rq).  fixing though as yes, very "off"

**Review by taoteg — APPROVED** [review]
So be it. =)

**Review by sophia-massie — APPROVED** [review]
LGTM!

**Pull request merged**